### PR TITLE
docs: improve language on 'Preventing default behavior'

### PR DESCRIPTION
**Pull request by neonpie**
improve language

# Overview

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests / types / typos

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [ ] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/docs/src/routes/tutorial/events/preventdefault/index.mdx [hunk 1]
```diff
@@ -3,13 +3,14 @@ title: Event Prevent Default
...
+For some events browsers have default behavior. For example, when you click a link, the browser will usually navigate to the link's `href`. There are times when we would like to prevent that. For this reason the `Event` API contains the `preventDefault()` method.
 
-Browser events are synchronous, but because Qwik is fine-grained the loadable Qwik execution model is asynchronous. This means that at the time the event is triggered, the event handler is not yet loaded. By the time the event is loaded the event has already been processed by the browser and calling `preventDefault()` will have no effect.
+Browser events are synchronous, but because Qwik is fine-grained the loadable Qwik the execution model is asynchronous. This means that at the time the event is triggered, the event handler is not yet loaded. By the time the event handler is loaded the event has already been processed by the browser and calling `preventDefault()` will have no effect.
```
Comment (zanettin) [inline packages/docs/src/routes/tutorial/events/preventdefault/index.mdx:11]: not sure about this extra `the` here 😄

File: packages/docs/src/routes/tutorial/events/preventdefault/index.mdx [hunk 2]
```diff
@@ -3,13 +3,14 @@ title: Event Prevent Default
...
 
-To solve this Qwik provides a declarative API to automatically call `preventDefault()` when the event is triggered. This is achieved by adding `preventdefault:<event-name>` attribute to the element. This allows the Qwikloader to synchronously call `preventDefault()` when the event is triggered.
+To solve this Qwik provides a declarative API to automatically call `preventDefault()` when the event handler is triggered. This is achieved by adding the `preventdefault:<event-name>` attribute to the element. This allows Qwikloader to synchronously call `preventDefault()` when the event handler is triggered.
```
Comment (zanettin) [inline packages/docs/src/routes/tutorial/events/preventdefault/index.mdx:13]: imo this section was okay before. the event is triggered which then gets handled by the event handler, right?

**Review by zanettin — COMMENTED** [review]
thanks again for the adjustments 🙏

File: packages/docs/src/routes/tutorial/events/preventdefault/index.mdx (hunk 1 above)
Comment (neonpie) [reply packages/docs/src/routes/tutorial/events/preventdefault/index.mdx:11]: woops, you're right :D

**Review by zanettin — APPROVED** [review]
thank you 🙏

**Pull request merged**